### PR TITLE
Distributed Systems: zero to ?

### PR DESCRIPTION
**Pull request by gcandal**
Distributed Systems: zero to ?
========================

* Speaker   : Gabriel Candal
* Available : Any time (preferably day 1/2) 
* Length    : 30m
* Language  : English

Description
-----------

One of the things that always excited me after having started to program for a living was how could systems be designed in a way that you could increase its capacity, not by making brilliant micro-optimizations that are only within reach of a few enlighted individuals or the most diligent benchmarkers, but just by Throwing Money at the Problem™ (as in: adding more machines). Having come to realise that in order to make it happen I should learn this thing called "Distributed Systems", I've started my journey (more of walk on the desert, actually) from zero to hero. Spoiler alert: the first step was understanding there is a lot more that can be achieved other than increasing the system's throughput, such as reduced latency or increased availability and storage capacity.

If you share this belief that some problems can be better solved (or at all) by having multiple machines collaborating and have some interest (or need) in knowing how could you do that, I'll try not to disappoint you. Far from having reached the "hero" stage, I believe I can supply you with enough tools to avoid making the same mistakes I've made or had to deal with by sharing concrete advice on what you should have in mind when reading about the subject, bootstrapping your first prototypes or building, tuning and extending systems that use this kind of stuff.

In case you don't share the same vision or are in the stage where you're still well served by a monolith, please let me show you what it's like on the other side and some of the challenges and opportunities there are.

Speaker Bio
-----------

Unfortunately I'm not one of those premature wonder kids who started hacking on their ZX Spectrum (have never seen one, to be honest) before they even knew how to read. However, during college I decided to make up for the lost time and started grinding through all technologies, paradigms and MOOCs (completed 24 of them in 3 years) that sounded like a challenge.

While doing all this I still had time to write a thesis on visual programming for probabilistic programming (not that anyone's using it, random forests and boosting algorithms train faster anyway), take a summer internship at Critical Manufacturing writing C/C++ on Solaris and donate some Python and Javascript to help PLATFORME International go from being a prototype to a well-established scale-up.

Bottom line: I'm 24, a Software Engineer, and at the moment I'm contributing to Feedzai's efforts on making machine learning at scale (do we still call it big data?) easier, more available and performant.

Links
-----

* Blog: http://nextcommit.github.io
* Company: http://feedzai.com
* Github: http://github.com/gcandal

### CODE REVIEW

**Comment by augustocravosilva** [comment]
:+1:

**Comment by opedromiranda** [comment]
👍🏻

**Comment by promgamer** [comment]
:+1:

**Comment by mpgon** [comment]
👍

**Comment by Joao-Sa** [comment]
👍
